### PR TITLE
skip printing the logs when program is in terminated state.

### PR DESCRIPTION
**Pull request by shiqizng**
script prints logs for all test cases that has non-zero return code. When the process got killed it  has a return code 15, so this caused verbose logging.

### CODE REVIEW

File: test/scripts/e2e_client_runner.py [hunk 1]
```diff
@@ -113,11 +113,22 @@ def _script_thread_inner(runset, scriptname):
     runset.running(scriptname, p)
     timeout = read_script_for_timeout(scriptname)
     try:
+        # Popen return code: https://docs.python.org/3/library/subprocess.html#subprocess.Popen.returncode
```
Comment (brianolson) [inline test/scripts/e2e_client_runner.py:116]: excessive comment; retcode=0 meaning success is standard unix for decades. We can drop these two new lines of comment.

File: test/scripts/e2e_client_runner.py [hunk 2]
```diff
@@ -113,11 +113,22 @@ def _script_thread_inner(runset, scriptname):
...
+        # 0 means success. A negative value -N indicates that the child was terminated by signal N (POSIX only).
         retcode = p.wait(timeout)
     except subprocess.TimeoutExpired as te:
         sys.stderr.write('{}\n'.format(te))
         retcode = -1
     dt = time.time() - start
+
+    if retcode != 0:
+        logger.debug("script file: ", scriptname, "retcode: ", retcode)
+
```
Comment (brianolson) [inline test/scripts/e2e_client_runner.py:126]: don't need this, there's retcode != 0 logging just below

**Review by brianolson — CHANGES_REQUESTED** [review]

File: test/scripts/e2e_client_runner.py (hunk 1 above)
Comment (shiqizng) [reply test/scripts/e2e_client_runner.py]: code updated.

File: test/scripts/e2e_client_runner.py (hunk 2 above)
Comment (shiqizng) [reply test/scripts/e2e_client_runner.py]: removed these lines.

**Review by brianolson — APPROVED** [review]
Thanks. I'm glad this turned out to be simple with most of the parts built.

**Review by algorandskiy — APPROVED** [review]
@shiqizng for the next PRs, please fork `algorand/go-algorand` repo and submit PRs from the fork.

**Pull request merged**